plugins/modules/java_cert.py

```diff
@@ -281,7 +281,8 @@ def _export_public_cert_from_pkcs12(module, executable, pkcs_file, alias, passwo
     (export_rc, export_stdout, export_err) = module.run_command(export_cmd, data=password, check_rc=False)
 
     if export_rc != 0:
-        module.fail_json(msg="Internal module failure, cannot extract public certificate from pkcs12, error: %s" % export_stdout,
+        module.fail_json(msg="Internal module failure, cannot extract public certificate from pkcs12, message: %s" % export_stdout,
```

Comment:
While we're at it, let's fix the capitalization:
```suggestion
        module.fail_json(msg="Internal module failure, cannot extract public certificate from PKCS12, message: %s" % export_stdout,
```